src/tools/chip-cert/BUILD.gn

```diff
@@ -37,7 +37,9 @@ executable("chip-cert") {
     "chip-cert.h",
   ]
 
-  public_configs = [ "${chip_root}/src/crypto:openssl_config" ]
+  if (chip_crypto == "openssl") {
```

Comment:
Do we need this change at all?

Reply:
Right, we don't build chip-cert in the release workflow, so we don't need it.